src/gt_frontend/py_to_gtscript.py

```diff
@@ -0,0 +1,225 @@
+# -*- coding: utf-8 -*-
+#
+# Eve Toolchain - GT4Py Project - GridTools Framework
+#
+# Copyright (c) 2020, CSCS - Swiss National Supercomputing Center, ETH Zurich
+# All rights reserved.
+#
+# This file is part of the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+import ast
+import enum
+import inspect
+import sys
+import typing
+
+import typing_inspect
+
+import eve.types
+import gtc.common
+from eve import UIDGenerator
+
+from . import ast_node_matcher as anm
+from . import gtscript_ast
+from .ast_node_matcher import Capture
+
+
+class PyToGTScript:
+    @staticmethod
+    def _all_subclasses(typ, *, module=None):
+        """
+            Return all subclasses of a given type. The type must be one of
+             - GT4PyAstNode (returns all subclasses of the given class)
+             - Union (return the subclasses of the united)
+             - ForwardRef (resolve the reference given the specified module and return its subclasses)
+             - built-in python type: str, int, type(None) (return as is)
```

Comment:
```suggestion
            Return all subclasses of a given type. 
            
            The type must be one of
            
            - GT4PyAstNode (returns all subclasses of the given class)
            - Union (return the subclasses of the united)
            - ForwardRef (resolve the reference given the specified module and return its subclasses)
            - built-in python type: str, int, type(None) (return as is)
```

Also this can be type annotated.

Reply:
Fixed